catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/JdbcCatalogOperations.java

```diff
@@ -510,4 +513,16 @@ public PropertiesMetadata topicPropertiesMetadata() throws UnsupportedOperationE
     throw new UnsupportedOperationException(
         "Jdbc catalog doesn't support topic related operations");
   }
+
+  public static void deregisterDriver(Driver driver) throws SQLException {
+    if (driver
+        .getClass()
+        .getClassLoader()
+        .getClass()
+        .getName()
+        .equals("com.datastrato.gravitino.utils.IsolatedClassLoader$1")) {
```

Comment:
Why there's a `$1`, can you use the canonical name? Compared to `$1` is not safe, it will possibly be changed for different JVMs.

Reply:
OK